docs/docs/api/DiagnosticsChannel.md

```diff
@@ -202,3 +202,60 @@ diagnosticsChannel.channel('undici:websocket:pong').subscribe(({ payload }) => {
   console.log(payload)
 })
 ```
+The below channels collectively act as [`tracingChannel.tracePromise`](https://nodejs.org/api/diagnostics_channel.html#tracingchanneltracepromisefn-context-thisarg-args) on `fetch`. So all of them will publish the arguments passed to `fetch`.
+
+## `undici:fetch:start`
+
+This message is published when `fetch` is called, and will publish the arguments passed to `fetch`.
+
+```js
+import diagnosticsChannel from 'diagnostics_channel'
+diagnosticsChannel.channel('undici:fetch:start').subscribe(({ input, init }) => {
```

Comment:
When using `channel(...)` rather than `tracingChannel(...)` you also need to include the `tracing:` prefix.